integration-tests/pom.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>testgrid-framework</artifactId>
+        <groupId>org.wso2.testgrid</groupId>
+        <version>0.9.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>org.wso2.testgrid.integration</artifactId>
+    <name>TG - Integration Tests</name>
```

Comment:
We can set the name as `WSO2 TestGrid Framework - Integration Tests` since we have used `WSO2 TestGrid Framework` prefix in all the poms

Reply:
This is Fixed